catadioptre-annotations/src/main/kotlin/io/aerisconsulting/catadioptre/kotlin/KotlinTestableProcessor.kt

```diff
@@ -277,4 +286,10 @@ internal class KotlinTestableProcessor : AbstractProcessor() {
         testableClassFile.addFunction(functionBuilder.build())
     }
 
+    private fun getLowestVisibility(parametersModifiers: MutableList<KModifier>, classVisibilityModifier: KModifier): KModifier {
+
+        val kModifierWithClassification = parametersModifiers.plus(classVisibilityModifier).map { KModifierClassification.getByKModifier(it) }
```

Comment:
(parametersModifiers + classVisibilityModifier).minBy{ it.ordinal } should be shorter as a unique statement of the method.